app/models/student.rb

```diff
@@ -284,7 +285,7 @@ def create_user_account
     user = User.new do |u|
       u.first_name = first_name
       u.last_name = last_name
-      u.username = admission_no
+      u.username = first_name
```

Comment:
for now it is not getting used but we might need it in future.
Ideally it should be unique.